.github/workflows/release_portable_linux_packages.yml

```diff
@@ -247,6 +247,20 @@ jobs:
               "rocm_version": "${{ needs.setup_metadata.outputs.version }}"
             }
 
+      - name: Trigger build JAX wheels
+        if: ${{ github.repository_owner == 'ROCm' }}
+        uses: benc-uk/workflow-dispatch@e2e5e9a103e331dad343f381a29e654aea3cf8fc # v1.2.4
+        with:
+          workflow: build_linux_jax_wheels.yml
+          inputs: |
+            { "amdgpu_family": "${{ matrix.target_bundle.amdgpu_family }}",
+              "python_versions": "3.12",
```

Comment:
Fixed Python version here? Should this use a matrix across versions somehow?

Reply:
The latest release of the JAX plugin supports Python 3.10, 3.11, and 3.12 for the wheel build, but we only use 3.10 and 3.12 in our Ubuntu docker image builds. 3.10 support is going to be dropped next release, so I just left it as 3.12.